src/ripple/rpc/tests/AccountObjects_test.cpp

```diff
@@ -0,0 +1,250 @@
+    "value": "1"
+  },
+  "TakerPays": "100000000",
+  "index":
+    "A984D036A0E562433A8377CA57D1A1E056E58C0D04818F8DFD3A1AA3F217DD82"
+})json"
+,
+R"json(
+{
+  "Account": "rPMh7Pi9ct699iZUTWaytJUoHcJ7cgyziK",
+  "BookDirectory":
+  "B025997A323F5C3E03DDF1334471F5984ABDE31C59D463525D038D7EA4C68000",
+  "BookNode": "0000000000000000",
+  "Flags": 65536,
+  "LedgerEntryType": "Offer",
+  "OwnerNode": "0000000000000000",
+  "Sequence": 5,
+  "TakerGets": {
+    "currency": "USD",
+    "issuer" : "r32rQHyesiTtdWFU7UJVtff4nCR5SHCbJW",
+    "value" : "1"
+  },
+  "TakerPays" : "100000000",
+  "index" :
+    "CAFE32332D752387B01083B60CC63069BA4A969C9730836929F841450F6A718E"
+}
+)json"
+};
+
+class AccountObjects_test : public beast::unit_test::suite
+{
+public:
+    void testErrors()
+    {
+        testcase("error cases");
+
+        using namespace jtx;
+        Env env(*this);
+
+        Json::Value params;
+        // test error on no account
+        {
+            auto resp = env.rpc("json", "account_objects", to_string(params));
+            expect(resp[jss::result][jss::error_message] ==
+                "Missing field 'account'.");
+        }
+        // test error on  malformed account string.
+        {
+            params[jss::account] = "n94JNrQYkDrpt62bbSR7nVEhdyAvcJXRAsjEkFYyqRkh9SUTYEqV";
+            auto const resp = env.rpc("json", "account_objects", to_string(params));
+            expect(resp[jss::result][jss::error_message] ==
+                "Disallowed seed.");
+        }
+        // account_info with an account that's not in the ledger.
+        {
+            params[jss::account] = Account{ "bogie" }.human();
+            auto const info = env.rpc("json", "account_objects", to_string(params));
+            expect(info[jss::result][jss::error_message] ==
+                "Account not found.");
+        }
+    }
+
+    void testUnsteppedThenStepped()
+    {
+        testcase("unsteppedThenStepped");
+
+        using namespace jtx;
+        Env env(*this);
+
+        Account const gw1{ "G1" };
+        Account const gw2{ "G2" };
+        Account const bob{ "bob" };
+
+        auto const USD1 = gw1["USD"];
+        auto const USD2 = gw2["USD"];
+
+        env.fund(XRP(1000), gw1, gw2, bob);
+        env.trust(USD1(1000), bob);
+        env.trust(USD2(1000), bob);
+ 
+        env(pay(gw1, bob, USD1(1000)));
+        env(pay(gw2, bob, USD2(1000)));
+
+        env(offer(bob, XRP(100), bob["USD"](1)),txflags(tfPassive));
+        env(offer(bob, XRP(100), USD1(1)), txflags(tfPassive));
+
+        Json::Value bobj[4];
+        for (int i = 0; i < 4; ++i)
+            Json::Reader{}.parse(bobs_account_objects[i], bobj[i]);
+
+        // test 'unstepped'
+        // i.e. request account objects without explicit limit/marker paging
+        {
+            Json::Value params;
+            params[jss::account] = bob.human();
+            auto resp = env.rpc("json", "account_objects", to_string(params));
```

Comment:
I'm guessing `env.rpc("account_objects", bob.human());`  would also work - still not clear to me if it's better/worse/indifferent to use the 'json' generic handler in the context of tests.